xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -244,10 +244,27 @@ This is the reason why the constructor accepts a <xref:System.Text.Json.JsonRead
         <Parameter Name="utf8Destination" Type="System.Span&lt;System.Byte&gt;" Index="0" FrameworkAlternate="net-7.0" />
       </Parameters>
       <Docs>
-        <param name="utf8Destination">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="utf8Destination">A buffer to write the unescaped UTF-8 bytes into.</param>
+        <summary>Copies the current JSON token value from the source, unescaped as a UTF-8 string to the destination buffer.</summary>
+        <returns>The number of bytes written to <paramref name="utf8Destination" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Unlike <xref:System.Text.Json.Utf8JsonReader.GetString>, this method does not support <xref:System.Text.Json.JsonTokenType.Null>.
+            
+             This method will throw <xref:System.ArgumentException> if the destination buffer is too small to hold the unescaped value.
+             An appropriately sized buffer can be determined by consulting the length of either <xref:System.Text.Json.Utf8JsonReader.ValueSpan> or <xref:System.Text.Json.Utf8JsonReader.ValueSequence>,
+             since the unescaped result is always less than or equal to the length of the encoded strings.
+
+          ]]></format>
+        </remarks>
+        <exception cref="T:System.InvalidOperationException">Thrown if trying to get the value of the JSON token that is not a string
+             (i.e. other than <see cref="F:System.Text.Json.JsonTokenType.String" /> or <see cref="F:System.Text.Json.JsonTokenType.PropertyName" />.
+             <seealso cref="P:System.Text.Json.Utf8JsonReader.TokenType" />
```

Comment:
This needs to be moved out of the exception tag.

Reply:
Resolving with this suggestion: https://github.com/dotnet/dotnet-api-docs/pull/8185/files#r914173079